static/js/i18n.js

```diff
@@ -43,3 +43,69 @@ export function $t_html(descriptor, values) {
         ),
     });
 }
+
+// This function replaces get_language_list_for_template
+// function used previously in the backend for getting data
+// in the desired form for the language list modal used the frontend.
```

Comment:
This comment is a bad pattern -- once we merge this commit, the old function won't exist in the codebase, so references to it will be useless.  Instead, we should write a comment explaining what this function does.

Reply:
Oh yes,right. Thanks.